src/systems/filecoin_markets/storage_market/storage_market_actor.id

```diff
@@ -1,66 +1,62 @@
 import actor "github.com/filecoin-project/specs/systems/filecoin_vm/actor"
 import addr "github.com/filecoin-project/specs/systems/filecoin_vm/actor/address"
-import block "github.com/filecoin-project/specs/systems/filecoin_blockchain/struct/block"
-import base_markets "github.com/filecoin-project/specs/systems/filecoin_markets"
-
-type StorageDeal base_markets.StorageDeal
-type StorageDealID base_markets.StorageDealID
+import deal "github.com/filecoin-project/specs/systems/filecoin_markets/deal"
 
 type StorageParticipantBalance struct {
     Locked     actor.TokenAmount
     Available  actor.TokenAmount
 }
 
-type PublishStorageDealSuccess struct {
-    StorageDealID
-}
-type PublishStorageDealError struct {}
-type PublishStorageDealResponse struct {
-    Success  PublishStorageDealSuccess
-    Error    PublishStorageDealError
-}
-
 type StorageMarketActor struct {
     // need access to:
     // - DataTransferSubsystem
     //   - transfer data
-    // - BlockchainSubsystem
-    //   - needs access to StateTree
-    //     to verify storage capacity and pledge amount
-    //     and slash storage providers for storage faults
-    //    - needs access to access to MessagePool if deals on chain
     // - NetworkSubsystem
     //   - needs access to MessagePubsub
 
-    ParticipantBalances  {addr.Address: StorageParticipantBalance}
-    StorageDeals         {StorageDealID: StorageDeal}
+    Balances  {addr.Address: StorageParticipantBalance}
+    Deals     {deal.DealID: deal.StorageDeal}
```

Comment:
Is this just a big map? Thats not going to scale at all

Reply:
potentially move this mapping to `StorageMinerActor` and `StorageMarketActor` only processes the balances cc @jbenet